### PR TITLE
Update acra-keys with symmetric_keys

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Updated `acra-keys` **read** to work with symmetric storage keys. 
Corresponding tests were added.

## Checklist

- [x] Change is covered by automated tests
- [x] The [coding guidelines] are followed
- [x] Public API has proper documentation in the [Acra documentation] site or has PR on [documentation repository] 
  with new changes
- [x] CHANGELOG.md is updated (in case of notable or breaking changes)
- [x] CHANGELOG_DEV.md is updated
- [x] Benchmark results are attached (if applicable)
- [x] [Example projects and code samples] are up-to-date (in case of API changes) 

[coding guidelines]: https://golang.org/doc/effective_go
[Example projects and code samples]: https://github.com/cossacklabs/acra-engineering-demo
[Acra documentation]: https://docs.cossacklabs.com/
[documentation repository]: https://github.com/cossacklabs/product-docs

### CODE REVIEW

File: cmd/acra-keys/keys/generate.go [hunk 1]
```diff
@@ -57,6 +55,7 @@ type GenerateKeyParams interface {
 	ZoneID() []byte
 	GenerateNewZone() bool
 	GenerateZoneKeys() bool
+	GenerateZoneSymKey() bool
```
Comment (Lagovas) [inline cmd/acra-keys/keys/generate.go:58]: We use full word `symmetric` everywhere. So lets use it here too -> `GenerateZoneSymmetricKey`.

Comment (ZhmakaAS) [reply cmd/acra-keys/keys/generate.go:58]: Got it, let me replace.

File: keystore/keystore.go [hunk 2]
```diff
@@ -281,6 +281,9 @@ type StorageKeyCreation interface {
 	// Generates a new key pair and replaces the current key pair for given zone ID.
 	// Returns new publie key data, error.
 	RotateZoneKey(zoneID []byte) ([]byte, error)
+	// Generates a new symetric key and replaces the current key for given zone ID.
+	// Returns new key data, error.
+	RotateSymZoneKey(zoneID []byte) error
```
Comment (Lagovas) [inline keystore/keystore.go:286]: this one better to rename too) to be consistent

**Review by Lagovas — APPROVED** [review]
lgtm. nice to see green tests)

**Pull request merged**